### ISSUE TITLE
Add CLI support for DIND

### ISSUE THREAD

**Issue by giorgiosav** [mlbench/mlbench-core#42]
Would be nice to have support for something local like DIND for testing purposes.

**Comment by ehoelzl** [comment]
I can take care of that, I already have a script that does it

**Comment by mmilenkoski** [comment]
Once this is done, maybe we should write a tutorial on how to set everything up for local development and testing, so that it is easier for new contributors to join the project.

**Comment by giorgiosav** [comment]
I believe we should close this given @mmilenkoski's work with Kubernets-in-Docker right?
